src/app/components/Input/styles.module.scss

```diff
@@ -0,0 +1,15 @@
+.content-input {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
use global help classes, check `layout.scss` file to know how to use.

for your case when you assign class `content-input` you append in the classname `row wrap`